specification/maps/data-plane/Route/preview/2023-10-01-preview/route.json

```diff
@@ -462,7 +462,7 @@
           "readOnly": true
         },
         "durationInSeconds": {
-          "description": "Estimated travel time in seconds property that includes the delay due to real-time traffic. Note that even when traffic=false travelTimeInSeconds still includes the delay due to traffic. If DepartAt is in the future, travel time is calculated using time-dependent historic traffic data.",
+          "description": "Estimated travel time in seconds property without traffic conditions. This value is provided if optimizeRoute does not include traffic considerations.",
```

Comment:
Estimated travel time in seconds that does not include delays on the route due to traffic conditions.